cmake/external_mkldnn.cmake

```diff
@@ -82,7 +82,7 @@ foreach(LIB ${MKL_LIBS})
 endforeach()
 
 set(MKLDNN_GIT_REPO_URL https://github.com/intel/mkl-dnn)
-set(MKLDNN_GIT_TAG "b9558fd")
+set(MKLDNN_GIT_TAG "21fb5f2af1dd14e132af4f1b79160977ee487818")
```

Comment:
Is the patch command still valid with this tag?

Reply:
This tag is still an RC. Should wait until the release tag is ready.

Reply:
Thanks @silee2 . Yes, will wait.